### DISCUSSION TITLE
Can the windy.com radar page be added easily to weewx-wdc?

### DISCUSSION THREAD

**Discussion by JonFear** [Daveiano/weewx-wdc#107] [Ideas]
Just come across your skin and love the simple easy style.

I see that you use the German system for your radar. As the Metoffice version is poor, I tend to use windy as my goto radar page. It is used in the Belchertown skin.

I have taken a quick look at the code and not being software literate I cannot see an easy way for me to add it.

Any thoughts? It maybe a useful addition for many people worldwide...

Jon

**Comment by Daveiano** [comment]
Thank you for your kind words!

I just had a look at Belchertown to understand your question. You are talking about the `radar_html` setting in the `[Extras]` section of the skin.conf in Belchertown, right? With this, users can set whatever they want as the radar image, for windy.com it is an embed (iFrame) Code, I guess!?

The current implementation of the radar image in the skin just shows an image, so the skin only accepts image URLs (and gifs of course) at the moment.

**Comment by JonFear** [comment]
Hi
Yes looking at the Belchertown skin, the radar does appear to be in an iFrame. Not being a programmer I do not know how difficult it would be to actually put this in place however I suspect that as Windy is worldwide, it could have quite a large audience. 
Thank you again for the skin, I love it and when we have the ability for the radar I will move it to my main online server...
Jon

**Comment by Daveiano** [comment]
Yes, this will be a useful addition for the skin, I created an issue for that: #108. This will be included in the next 3.0.0 update!

Thank you again!

**Comment by JonFear** [comment]
Thank you so much!

Very exciting. Just setting up the DS forecast...

Jon

**Comment by JonFear** [comment]
Thank you for the updates, house keeping now by closing subject.
